### PR TITLE
feat: Adding icons next to headings to have summarized info

### PR DESCRIPTION
**Pull request by shiprarawal**

### CODE REVIEW

**Comment by alisonhall** [comment]
Hey @shiprarawal , do you think you could add some screenshots to this PR? I was also wondering if there's any way to optimize the code a bit so that we don't have as many nested loops. Otherwise it looks pretty good!

**Comment by shiprarawal** [comment]
Hi @alisonhall ,
Please find the screenshots below : 
Screenshot 2024-06-07 at 10 08 59 AM
Screenshot 2024-06-07 at 10 09 28 AM
Screenshot 2024-06-07 at 10 09 39 AM
 Let me take a look for code optimization. Thanks!

**Comment by shiprarawal** [comment]
Hi @alisonhall , I have updated the code . Please take a look.

**Comment by alisonhall** [comment]
@shiprarawal , this code is much simpler, thanks! I just added 2 minor comments/questions.

**Comment by zoltan-dulac** [comment]
I am okay with this as long as @alisonhall is.  Please merge when you get @alisonhall 's approval.

File: js/global.js [hunk 1]
```diff
@@ -62,6 +62,35 @@ function buildFlyoutMenuHTML() {
...
+        // Iterate over siblings with the specified class name
+        while (nextElement && nextElement.classList.contains(className)) {
+            const imgElements = nextElement.querySelectorAll('img');
+
+            // Append attributes of img elements to the images string
+            imgElements.forEach((img) => {
+                const attributes = Array.from(img.attributes)
+                    .map((attr) => {
+                        return attr.name !== 'class'
+                            ? `${attr.name}="${attr.value}"`
```
[the remark this answers is not in the document]
Comment (shiprarawal) [reply js/global.js:81]: Hi Alison,
As img tag doesn't have any innerHTML, so the intention here is to iterate through all the attributes and then add them to image tag. Here, 'attr' attributes returns the attribute object. We want to use the name and value properties so we can set it in the image tag like . So, writing attr will no help.

File: js/global.js [hunk 2]
```diff
@@ -62,6 +62,35 @@ function buildFlyoutMenuHTML() {
...
+                            : ((attr.value += ' enable-stats__heading-icon'),
+                              `${attr.name}="${attr.value}"`);
```
[the remark this answers is not in the document]
Comment (shiprarawal) [reply js/global.js:83]: It's like doing multiples actions using ternary operators. This is one of the articel for more information: https://simon-ugorji.medium.com/execute-multiple-actions-in-a-ternary-block-5f7ce5130e0c

[the remark this answers is not in the document]
Comment (alisonhall) [reply js/global.js:83]: Thank you!

**Review by alisonhall — APPROVED** [review]

**Pull request merged**